pkg/controller/grafanadashboard/dashboard_controller.go

```diff
@@ -175,8 +175,8 @@ func (r *ReconcileGrafanaDashboard) checkNamespaceLabels(dashboard *grafanav1alp
 }
 
 func (r *ReconcileGrafanaDashboard) reconcileDashboards(request reconcile.Request, grafanaClient GrafanaClient) (reconcile.Result, error) {
-	// Collect known and namespace dashboards
-	knownDashboards := r.config.GetDashboards(request.Namespace)
+	// Collect known and namespace dashboards, cluster wide search with ""
+	knownDashboards := r.config.GetDashboards("")
```

Comment:
What's the reason for doing a cluster-wide search now?

Reply:
Probably should be removed if we don't want to look in all namespaces by default (we definitely don't) when I changed this I was just cutting out the abstraction from the previous `request.Namespace`, where the initial request is an empty string anyway. although there still might be some other reasons of which i'm not aware, I haven't noticed much of a difference between doing `""` and `request.namespace`

Reply:
On second thought this is actually needed, and that's because if we don't do a cluster wide search we won't be able to delete the dashboard from our grafana instance
as that initial cluster wide request gets all dashboards and reconciles our `grafana-namespaced` `grafana instance`, and will only be able to delete dashboards from the one it's running in
so we constantly have to update our knowndashboards list (from all namespaces)
without that we'll only be ever able to update dashboards in our grafana instance when we relaunch it (i.e when we send out that initial cluster-wide search request)